aptos-move/e2e-tests/src/executor.rs

```diff
@@ -395,13 +380,7 @@ impl FakeExecutor {
         &self,
         txn_block: Vec<Transaction>,
     ) -> Result<Vec<TransactionOutput>, VMStatus> {
-        let (result, _) = ParallelAptosVM::execute_block(
-            txn_block,
-            &self.data_store,
-            usize::min(4, num_cpus::get()),
-        )?;
-
-        Ok(result)
+        ParallelAptosVM::execute_block(txn_block, &self.data_store, num_cpus::get())
```

Comment:
please keep the usize::min

Reply:
damn, thanks for the catch. no idea what happened, maybe I messed up the rebase. I remember this is the optimization for tests.
Will also do the other check later than if block is small use sequential.